src/NBitcoin/Network.cs

```diff
@@ -146,6 +146,7 @@ public class ConsensusOptions
         }
 
         public ConsensusOptions Options { get; set; }
+        public NetworkOptions NetworkOptions { get; set; } = NetworkOptions.TemporaryOptions;
```

Comment:
This is probably wrong, we should do the same as all other networks and when building the network set the options.
Even add an overload the the NetworkBuilder to set Options.
If this is temporary and you will change it in the next PR that's fine.

Reply:
It is temporary.